pkg/service/catalog.go

```diff
@@ -158,6 +162,7 @@ func NewService(
 	serviceManager ServiceManager,
 	plans ...Plan,
 ) Service {
+	azureConfig, _ := azure.GetConfigFromEnvironment()
```

Comment:
@krancour this seemed like the cleanest way to grab this. I didn't want to bubble this through the rest of the framework and didn't think there was a better way to do this. open to suggestions.